From mfuzzey@gmail.com  Wed Oct  6 13:39:37 2010
From: Martin Fuzzey <mfuzzey@gmail.com>
Subject: USB: usbtest: Support gadget zero isochronous configurations
To: Greg KH <greg@kroah.com>
Cc: linux-usb@vger.kernel.org
Date: Wed, 06 Oct 2010 17:43:46 +0200
Message-ID: <20101006154346.26831.55484.stgit@srv002.fuzzey.net>

Gadget zero now provides isochronous endpoints via f_iso;
add support for this to usbtest.

Signed-off-by: Martin Fuzzey <mfuzzey@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/misc/usbtest.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/usb/misc/usbtest.c
+++ b/drivers/usb/misc/usbtest.c
@@ -2083,7 +2083,8 @@ static struct usbtest_info gz_info = {
 	.name		= "Linux gadget zero",
 	.autoconf	= 1,
 	.ctrl_out	= 1,
-	.alt		= 0,
+	.alt		= -1,
+	.iso		= 1,
 };
 
 static struct usbtest_info um_info = {
